Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -4455,9 +4457,12 @@ private function findListObjectsOnAllStacksAndSubstacks pCallback, pCallbackTarg
    
```

Comment:
Same comment as above applies for the mainstacks variable. Perhaps we need an `ideMainStacks pIncludeIDEStacks` if returning them based on gRevDevelopment is common.